src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_INT = 10;
```

Comment:
It would be good to have a separate constant for the default circle radius and max value of figure properties
so we can change them separately